spec/rails_helper.rb

```diff
@@ -5,6 +5,7 @@
 # Prevent database truncation if the environment is production
 abort('The Rails environment is running in production mode!') if Rails.env.production?
 require 'rspec/rails'
+require 'capybara/rspec'
 # Add additional requires below this line. Rails is not loaded until this point!
 
 # Requires supporting ruby files with custom matchers and macros, etc, in
```

Comment:
- [ ] Good job trying to add some of changes required by the previous reviewer. So, I noticed that your tests cannot run because of `line 35` in `spec/rails_helper.rb` file. You need to comment out that line since you don't have `spec/fixtures` directory.
- [ ] After fixing the above issue you should make sure all your tests are passing as tose failing tests are part of the project requirements and they have to pass before you move on.
![Screenshot from 2023-01-31 13-02-44](https://user-images.githubusercontent.com/83042742/215743105-56f49712-a05d-4316-9d50-fa2c88109b4b.png)